src/luascript.cpp

```diff
@@ -578,21 +586,28 @@ void LuaScriptInterface::pushVariant(lua_State* L, const LuaVariant& var)
 	lua_createtable(L, 0, 2);
 	setField(L, "type", var.type);
 	switch (var.type) {
-		case VARIANT_NUMBER:
+		case VARIANT_NUMBER:{
```

Comment:
I think something is missing here

Reply:
Thank you, I will send revision 3 tomorrow.